commands/apps.go

```diff
@@ -293,49 +305,74 @@ func (c *AppsInitCmd) InitializeNodeBasicApp(stdout, stderr bytes.Buffer) (err e
 	checkPkgJSON, err := os.Open("package.json")
 	if err != nil {
 		log.Println("[WARN] package.json does not exist. Creating package.json")
-		cmd := exec.Command("npm", "init", "-y")
-		cmd.Stdout = &stdout
-		cmd.Stderr = &stderr
-		err := cmd.Run()
+		err := c.CreatePkgJson(stdout, stderr)
 		if err != nil {
 			log.Println("[ERROR] There was an error creating package.json. Is node installed?")
-			panic(err)
+			return fmt.Errorf("there was an error creating package.json. Is node installed? %w", err)
 		} else {
 			log.Println("[INFO] package.json created")
 		}
 	}
 	defer checkPkgJSON.Close()
 	validPkgJSON, err := os.OpenFile("package.json", os.O_RDWR, 0777)
 	if err != nil {
-		panic(err)
+		return fmt.Errorf("failed to open package.json %w", err)
 	}
+	defer validPkgJSON.Close()
 	log.Println("[INFO] Validating package.json")
-	finfo, err := validPkgJSON.Stat()
+	buf, err := ioutil.ReadFile("package.json")
```

Comment:
Rename with `os.ReadFile`. `ioutil` is deprecated in 1.16 which was released yesterday.